build_and_run.sh

```diff
@@ -21,11 +21,22 @@
 
 set -eu
 
-readonly unzip_dir="${1:-$HOME/Applications}"
+unzip_dir=${unzip_dir:-$HOME/Applications}
```

Comment:
I'm not sure there's value in respecting the unzip_dir or bazel_path ENV vars, maybe just make those the default values?